packages/next/src/client/components/router-reducer/reducers/server-action-reducer.ts

```diff
@@ -220,7 +220,8 @@ export function serverActionReducer(
           // TODO-APP: remove ''
           [''],
           currentTree,
-          treePatch
+          treePatch,
+          true
```

Comment:
optional, instead of a boolean param, could we perhaps just split the function into two so that it's more explicit? 

something like eapplyRouterStateRevalidationPatchToTree (but not this)